.github/workflows/scorecard.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   analysis:
     name: Scorecard analysis
     runs-on: ubuntu-latest
-    if: github.repository == 'llvm/llvm-project'
+    if: github.repository == 'intel/llvm'
```

Comment:
Should we update the link at line 5?

Reply:
yes we should :) I totally missed that part. TY